pkg/virt-controller/services/template.go

```diff
@@ -991,6 +991,10 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		},
 	}
 
+	if vmi.Spec.PriorityClassName != "" {
+		pod.Spec.PriorityClassName = vmi.Spec.PriorityClassName
```

Comment:
if we go with this api addition, we'll need  a unit test case in template_test.go that cross validates the priorityclass gets set on the pod as expected.

Reply:
I added test case. Is this enough ?

Reply:
yep, looks good :+1: